### DISCUSSION TITLE
Communication Plan

### DISCUSSION THREAD

**Discussion by kubraaksux** [bounswe/bounswe2023group1#37] [Q&A]
What is your suggestions about that? I asked to use slack but some dont. Seems like we need to take team decisions there. 
I checked out last year repositories. Sharing their communication plans there. 

<img width="848" alt="Screenshot 2023-03-11 at 17 32 00" src="https://user-images.githubusercontent.com/67320250/224491476-84466097-6f8f-4d1d-90ae-5a6c17a54294.png">

<img width="774" alt="Screenshot 2023-03-11 at 17 29 52" src="https://user-images.githubusercontent.com/67320250/224491523-d798abb3-d021-430c-8ea3-99c447f0af81.png">
<img width="865" alt="Screenshot 2023-03-11 at 17 30 24" src="https://user-images.githubusercontent.com/67320250/224491528-acf319b2-8a21-4763-92ee-774be2eb1dc3.png">
<img width="848" alt="Screenshot 2023-03-11 at 17 32 00" src="https://user-images.githubusercontent.com/67320250/224491531-0acc2273-5007-49a7-91cc-83a019f61a69.png">
<img width="866" alt="Screenshot 2023-03-11 at 17 32 57" src="https://user-images.githubusercontent.com/67320250/224491533-bd5cd4ab-e4d1-4e16-ad69-b809898e7b51.png">
<img width="851" alt="Screenshot 2023-03-11 at 17 33 33" src="https://user-images.githubusercontent.com/67320250/224491534-bffcc1d0-a36c-4d2d-a6d9-5bd42e49b781.png">
<img width="843" alt="Screenshot 2023-03-11 at 17 34 43" src="https://user-images.githubusercontent.com/67320250/224491536-ed9b9402-3097-40d5-a4ef-59a78daab24d.png">
<img width="905" alt="Screenshot 2023-03-11 at 17 35 06" src="https://user-images.githubusercontent.com/67320250/224491537-0e08acd3-8499-443c-a910-23f8ff1f8052.png">

**Comment by HarunErgen** [comment]
Are these headlines adequate for our communication plan? 
| Participants | Channel | Location | Time | Objective |
|--------------|----------|-----------|-------|------------|

**Reply by kubraaksux** [reply]
do you mean purpose when you say objective? and maybe we can add meeting type like urgent, weekly, any time, customer, additional or project?

**Reply by alitpc25** [reply]
I think we can add face-to-face, Discord, and WhatsApp for now and we may update it in the future. Also, we should add our communication plan to the sidebar, too.

**Reply by alperenDagi** [reply]
I agree with ali, current plan can be f2f, discord and whatsapp. I will be updating our discord server according to our needs.
